build.gradle

```diff
@@ -3,11 +3,11 @@ plugins {
   id "com.github.mxenabled.binks" version "0.0.1"
   id "com.github.mxenabled.vogue" version "1.0.1"
   id "com.github.mxenabled.coppuccino" version "3.1.2" apply false
-  id "com.github.mxenabled.hush" version "2.3.0" apply false
+  id "com.github.mxenabled.hush" version "2.3.1" apply false
   id "io.freefair.lombok" version "6.5.1" apply false
 }
 
-version "0.0.2"
+version "0.0.3"
```

Comment:
Not sure why this is showing up in this PR, as master is already at 0.0.3. Must be from my rebase.